publish-oci-reference.md

```diff
@@ -0,0 +1,35 @@
+# OCI image publishing reference
+
+This document lists the properties needing to [publish OCI images](publish-oci.md). It is part of a [container workshop](README.md), which details fundamental workflows for using .NET in containers.
+
+## Enable publishing
+
+OCI image publishing is not enabled by default with `dotnet publish`. Most users do not want to generate a container image every time they publish their project, but an app they can run directly.
+
+Container publishing can be enabled with one of the following gestures:
+
+- `PublishContainer` -- Include this target with `-t:PublishContainer` on the commandline.
+- `PublishProfile` -- Set this property with `-p PublishProfile=DefaultContainer` on the commandline or in a project file. This property is only available to ASP.NET Core apps.
+
+Console apps additionally require installing a NuGet package.
+
+```bash
+$ dotnet add package Microsoft.NET.Build.Containers --version 8.0.100-rc.2.23480.5
```

Comment:
By the time next Tuesday rolls around the stable 8.0.100 version of this package will be released, so the `--version` option won't be required. Should we preemptively remove it?

Reply:
Yup. The whole doc needs to be updated when we ship to remove all the RC2-ism. I plan to do that before our talk.